assets/js/admin/facebook-for-woocommerce-products-admin.js

```diff
@@ -370,6 +369,151 @@ jQuery( document ).ready( function( $ ) {
 		}
 
 
+		/**
+		 * Store the original value of the given element for later use
+		 *
+		 * @since 2.1.4-dev.1
```

Comment:
Also, since 2.1.4 is already out, we will have to update all of these to `2.1.5-dev.1`.